site/content/docs/developing/sidecars.en.md

```diff
@@ -74,6 +74,21 @@ sidecars:
         path: '/etc/mount1'
 ```
 
+Below is an example of specifying docker build arguments for a sidecar container.
+
+```yaml
+sidecars:
+  nginx:
+    image:
+      build:
+        dockerfile: path/to/dockerfile
+        context: context/dir
+        target: build-stage
+        cache_from:
+          - image: tag
+        args: value
+```
```

Comment:
nit: maybe we can remove this section from the doc but instead update the first example above with:

```yaml
sidecars:
  nginx:
    port: 80
    image: 
       build: src/reverseproxy/Dockerfile
    variables:
       NGINX_PORT: 80
```